app/src/main/java/uz/direction/coroutineshomework/MainActivity.kt

```diff
@@ -1,28 +1,42 @@
 package uz.direction.coroutineshomework
 
 import android.os.Bundle
+import android.util.Log
 import androidx.appcompat.app.AppCompatActivity
+import kotlinx.coroutines.*
 
 class MainActivity : AppCompatActivity() {
 
+    private lateinit var job: Job
     private lateinit var memesPresenter: MemesPresenter
 
+    val myCoroutineScope =
```

Comment:
It is not necessary for scope here as presenter already runs in a scope its logics